QRCoder/QRCodeGenerator.cs

```diff
@@ -1009,7 +1009,7 @@ private static BitArray PlainTextToBinaryNumeric(string plainText)
         for (int i = 0; i < plainText.Length - 2; i += 3)
         {
             // Parse the next three characters as a decimal integer.
-#if NET5_0_OR_GREATER
+#if NETCOREAPP2_1_OR_GREATER || NETSTANDARD2_1
```

Comment:
```suggestion
#if NETCOREAPP || NETSTANDARD2_1
```
I thinks this could be shortened for better legibility.
Older .NET Core targets than 2.1 aren't present. And .NET (5+) is covered by that compiler constant too.

Reply:
Well, as written:
- it matches the rest of the codebase
- it matches the conditions for which Span is supported

So I'd prefer to leave it as-is here for those reasons.  Perhaps it would be better to have a compile time constant defined for span support - similar to SYSTEM_DRAWING we have SPAN - then in the csproj we define it for the applicable target frameworks.  Then we change all uses to match.